lib/open_project/xls_export/hooks/work_package_hook.rb

```diff
@@ -2,12 +2,22 @@ module PrintableIssues
   class IssueHook  < Redmine::Hook::ViewListener
     # Add XLS format link below issue list
     def view_work_packages_index_other_formats(context)
```

Comment:
Never mind, it is indeed hard-coded within the [core](https://github.com/opf/openproject/blob/dev/app/controllers/api/experimental/work_packages_controller.rb#L92).